src/common/styles.css

```diff
@@ -2,16 +2,3 @@
 .assisted-ui-float-right {
   float: right !important;
 }
```

Comment:
Can we get rid of this one too?

Reply:
I'm not sure what the _pf-u-float-right does not work in all flows_ refers to...

Reply:
But it doesn't seem we are using the `inlineRight` option anywhere so maybe we could get rid of this option entirely?

Reply:
Any chance you can test it quickly?

Reply:
Tested. Looked fine to me.